shell/platform/fuchsia/flutter/tests/fakes/scenic/fake_flatland.cc

```diff
@@ -7,12 +7,16 @@
 #include <zircon/types.h>
 
 #include <algorithm>  // For std::remove_if
+#include <cmath>
 #include <memory>
 
 #include "flutter/fml/logging.h"
 
 namespace flutter_runner::testing {
 
+const fuchsia::math::Rect kUnclippedBounds = {-INT_MAX / 2, -INT_MAX / 2,
```

Comment:
Could we add a comment explaining what setting the bounds to [-INT_MAX/2, INT_MAX] does? I'm wondering why it's `-INT_MAX/2` but `INT_MAX`.

Reply:
The first 2 are the origin points of the rect, and the last 2 are its size. This has it so that the rects width/height is INT_MAX and is centered at the origin.

But I actually just updated that (along with added comment) in my latest commit because it should actually have its top-left corner at the origin. So new layout is (0,0, INT_MAX, INT_MAX) - sorry for confusion.